backend/src/test/service/CachingService.test.ts

```diff
@@ -0,0 +1,106 @@
+import { CachingService } from '../../main/service/CachingService';
+import { CachingObject } from '../../main/data/CachingObject';
+import { Promotion } from '../../main/entity/Promotion';
+import { DiscountType } from '../../main/data/DiscountType';
+import { User } from '../../main/entity/User';
+import redisMock from 'redis-mock';
+import { UserFactory } from '../factory/UserFactory';
+import { PromotionFactory } from '../factory/PromotionFactory';
+import { DiscountFactory } from '../factory/DiscountFactory';
+import { ScheduleFactory } from '../factory/ScheduleFactory';
+
+describe('tests for redis cache', function () {
+  let cachingService: CachingService;
+
+  beforeAll(() => {
+    const redisMockClient = redisMock.createClient();
+    cachingService = new CachingService(redisMockClient);
+  });
+
+  afterAll(() => {
+    const client = cachingService.getClient();
+    client.quit();
+  });
+
+  test('caching and getting lat/lon values', () => {
+    return cachingService
+      .cacheLatLonValues('ChIJIfBAsjeuEmsRdgu9Pl1Ps48', -34.2, -46.123)
```

Comment:
What do you think about using constants throughought this test class
```suggestion
      .cacheLatLonValues(TEST_PLACE_ID, TEST_LAT, TEST_LON)
```